ci/format_pre.sh

```diff
@@ -7,6 +7,7 @@
 
 FAILED=()
 CURRENT=""
+BASH_ERR_PREFIX="##[error]: "
```

Comment:
comment why this syntax?

The reason is that someday maybe we'll switch CI mechanisms (again) and want a different prefix for whatever the new system is, but won't know whether this is changeable.

Reply:
apologies - added